engine/OthelloEngine.py

```diff
@@ -0,0 +1,100 @@
+
+class GameEngine:
+   # white_team_file will be the file name of the white team's AI file
+   # black_team_file will be the file name of the black team's AI file
+   # output_file is the name of the file the game will be recorded to
+   # nxn is the size of the board. Default is 8x8
+   # time_limit is the amount of time for each turn in seconds
+   def init(self, white_team_file, black_team_file, output_file, n = 8, time_limit = 2.0):
```

Comment:
This should probably be `def __init__(...)`

Reply:
Good catch